### PR TITLE
[T.F 2.0 API Docs] Examples for exp, expm1 and equal math functions

### PR DESCRIPTION
**Pull request by SSaishruthi**
Reference Issue: #25802

This PR adds examples to the following math functions.

1. equal
2. exp
2. expm1

Thanks in advance.

### CODE REVIEW

File: tensorflow/core/api_def/base_api/api_def_Equal.pbtxt [hunk 1]
```diff
@@ -4,5 +4,13 @@ op {
   description: <<END
 *NOTE*: `Equal` supports broadcasting. More about broadcasting
 [here](http://docs.scipy.org/doc/numpy/user/basics.broadcasting.html)
+```python
+x = tf.constant([2, 4])
+y = tf.constant(2)
+tf.math.equal(x, y) ==> array([ True, False])
+
+x = tf.constant([2, 4])
+y = tf.constant([2,4])
```
Comment (martinwicke) [inline tensorflow/core/api_def/base_api/api_def_Equal.pbtxt]: Please add space after comma

File: tensorflow/core/api_def/base_api/api_def_Equal.pbtxt [hunk 2]
```diff
@@ -4,5 +4,13 @@ op {
   description: <<END
 *NOTE*: `Equal` supports broadcasting. More about broadcasting
 [here](http://docs.scipy.org/doc/numpy/user/basics.broadcasting.html)
+```python
```
Comment (martinwicke) [inline tensorflow/core/api_def/base_api/api_def_Equal.pbtxt]: I think you need a blank line here to make the renderer happy.

File: tensorflow/core/api_def/base_api/api_def_Exp.pbtxt [hunk 3]
```diff
@@ -1,4 +1,30 @@
 op {
   graph_op_name: "Exp"
   summary: "Computes exponential of x element-wise.  \\\\(y = e^x\\\\)."
+  description: <<END
+  This function computes exponential of every element in the input tensor.
```
Comment (martinwicke) [inline tensorflow/core/api_def/base_api/api_def_Exp.pbtxt]: *the* exponential

File: tensorflow/core/api_def/base_api/api_def_Exp.pbtxt [hunk 4]
```diff
@@ -1,4 +1,30 @@
...
+  tf.math.exp(x) ==> 7.389056
+
+  x = tf.constant([2.0, 8.0])
+  tf.math.exp(x) ==> array([7.389056, 2980.958], dtype=float32)
+  ```
+
+  For complex numbers, say 1+1j, real part is calculated like above ( e^(1) = 2.7182818284590).
+  Imaginary part e^(1j) will be calculated as, cos 1 + i sin 1 (e^ai = cos a + i sin a) which is
+  0.54030230586+0.8414709848j.
+
```
Comment (martinwicke) [inline tensorflow/core/api_def/base_api/api_def_Exp.pbtxt]: Please put all equations in backticks to make them render as code. This is very hard to read without that sort of font-based highlighting.

The math in this example is wrong. The example text is confusing and misleading, it needs to be rewritten.

This is the correct trigonometric replacement:  e^(x+iy) = e^x * e^iy = e^x * (cos y + i sin y).

File: tensorflow/core/api_def/base_api/api_def_Expm1.pbtxt [hunk 5]
```diff
@@ -2,6 +2,19 @@ op {
...
+  `e` denotes Euler's number and is approximately equal to 2.718281.
+
+  ```python
+  x = tf.constant(2.0)
+  tf.math.expm1(x) ==> 6.389056
+
+  x = tf.constant([2.0, 8.0])
+  tf.math.expm1(x) ==> array([6.389056, 2979.958], dtype=float32)
+
+  #Refer `exp` for explanation
```
Comment (martinwicke) [inline tensorflow/core/api_def/base_api/api_def_Expm1.pbtxt]: "Please refer to `exp` for an explanation."

Do you intend to add this kind of note to all functions which operate on complex numbers? Almost all functions do. I'm not sure we have to teach all our users about complex numbers. If they use them, they likely know how they work. If they don't, there's not much these examples will do to help them.

If we do want to add a uniform not and explanation about complex numbers, the docstring for exp is probably not the best place for it. I'd prefer to link out to wikipedia and keep the explanations here to a minimum.

**Review by martinwicke — CHANGES_REQUESTED** [review]

**Comment by SSaishruthi** [comment]
Hi @martinwicke 
I have made changes to the branch. I thought of pointing users about handling complex numbers in exponential function under `expm1`. That is the reason for adding the below line.

```
Refer `exp` for an explanation.
```

I have removed it for now. If you feel it is ok to add, I can make changes to it and push again.

**Comment by SSaishruthi** [comment]
@martinwicke Completed changes. Please let me know if this looks good.

File: tensorflow/core/api_def/base_api/api_def_Expm1.pbtxt [hunk 6]
```diff
@@ -2,6 +2,18 @@ op {
   graph_op_name: "Expm1"
   summary: "Computes exponential of x - 1 element-wise."
```
Comment (martinwicke) [inline tensorflow/core/api_def/base_api/api_def_Expm1.pbtxt]: Actually, can you change this summary? I think it is ambiguous at best. 

Maybe better: "Computes `exp(x)` - 1 element-wise."

**Comment by martinwicke** [comment]
Looks good after one more.comment.

**Comment by SSaishruthi** [comment]
> Looks good after one more.comment.

@martinwicke Updated. Thanks a lot

**Review by martinwicke — APPROVED** [review]

**Pull request merged**